### DISCUSSION TITLE
Sideband APK Sharing

### DISCUSSION THREAD

**Discussion by OtisByron** [markqvist/Reticulum#48] [Sideband]
One of the features I appreciated about the [Serval Mesh](http://servalproject.org/) app was the ability to share the app apk (via WiFi hotspot or Bluetooth) with someone who did not have the app. If not already on the Sideband roadmap it may be worth considering the ability to share the apk.

**Comment by samuk** [comment]
https://github.com/marcosdiez/shareviahttp might have some useful code if you do decide to build this into the .apk

**Comment by markqvist** [comment]
Very useful feature. This is a great idea, let's add it. If anyone wants to have a go at it, it would be awesome. Otherwise I will take a shot at it for the next release of Sideband. It'll be some days before I return to Sideband though, as I am currently working on the 0.3.5 release of Reticulum.
